### ISSUE TITLE
Add player events for the SDK

### ISSUE THREAD

**Issue by olavra** [decentraland/unity-renderer#300]
- [x] Event when player position or rotation changes
- [ ] ~~Event when player changes wearables~~ (derived to: #435)
- [x] Event when player does an emote

**Comment by kuruk-mm** [comment]
I had some problems with this issue that I'm working on out of the scope of the task.

First of all, the emote are called by a observable named `avatarMessageObservable`
That observable can notify an event of type `AvatarMessageType.USER_EXPRESSION`, but the problem was that the `uuid` of each emote event was totally randomized (Creating a new AvatarEntity on each played emote!).
So reviewing the problem further I discover that the `uuid` was incorrectly randomized on local. And for remote events was the `peerId` (2 letters unique id used for P2P Communication). Talking with @menduz we tried to modify it to use the `peerId` instead of the `random uuid` but internaly the system uses the `userId` to reference it own avatar, and the `peerId` to reference the remote avatars. So it's inconsistent.

I decided to use as it is, just modifying the `random uuid` for a `userId` and let a comment to review the inconsistency in the future.

Then, I tried to notify when a player changes a weareble. I'm working on that, I asked help to @nchamo, and with @pablitar and they said where to put the event.
